src/IssueViz.Security/HotspotsList/TableDataSource/HotspotTableEntry.cs

```diff
@@ -19,14 +19,13 @@
  */
 
 using System;
-using System.Windows;
 using Microsoft.VisualStudio.Shell.TableControl;
+using Microsoft.VisualStudio.Shell.TableManager;
 using SonarLint.VisualStudio.IssueVisualization.Models;
-using ImageMoniker = Microsoft.VisualStudio.Imaging.Interop.ImageMoniker;
 
 namespace SonarLint.VisualStudio.IssueVisualization.Security.HotspotsList.TableDataSource
 {
-    internal class HotspotTableEntry : IWpfTableEntry
+    internal class HotspotTableEntry : ITableEntry
```

Comment:
As we discussed offline, I'm having trouble with `IWpfTableEntry` interface due to method `bool TryCreateImageContent(string columnName, bool singleColumnView, out ImageMoniker content);`. 
`ImageMoniker` is in `Microsoft.VisualStudio.Imaging.Interop.14.0.DesignTime` assembly, but I couldn't get it to work: the MEF import just kept failing on missing assembly
```Microsoft.VisualStudio.Composition.PartDiscoveryException: Failure while scanning type "SonarLint.VisualStudio.IssueVisualization.Security.HotspotsList.TableDataSource.HotspotsTableDataSource". ---> Microsoft.VisualStudio.Composition.PartDiscoveryException: Part discovery failed at member tableEntries. ---> System.IO.FileNotFoundException: Could not load file or assembly 'Microsoft.VisualStudio.Imaging.Interop.14.0.DesignTime, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' or one of its dependencies. The system cannot find the file specified.
```
For now I changed it to `ITableEntry` and it works, but it doesn't support icons or tooltips. Once we need those, I'll invest more into the assembly problem.

Reply:
Ok. It's interesting that the missing assembly has `DesignTime` in the name; perhaps it contains interop types that should be embedded in our assembly.

Reply:
Just an observation: this seems like a logically separable change; I'd have probably kept it separate from the new interface implementation. It's not worth changing now though.

Reply:
Hmm, your comment about interop reminded me something: IssueViz imports `Microsoft.VisualStudio.Shell.Interop.14.0.DesignTime`, which I tried to import here and it didn't work either, but it also imports `Microsoft.VisualStudio.SDK.EmbedInteropTypes` -- and this might be the missing piece. I'll try it and see if it fixes it.